src/models/identity-provider.ts

```diff
@@ -0,0 +1,17 @@
+import Joi from 'joi';
+
+interface IdentityProvider {
+  hashId: string;
+  name: string;
+  url: string;
+  ssoClientId: string;
+}
+
+const schema = (): Joi.ObjectSchema => Joi.object().keys({
+  hashId: Joi.string().required().example('wasd2'),
+  name: Joi.string().required(),
+  url: Joi.string().required().example('https://login.microsoftonline.com/abvsd23fasdfasdf/v2.0/.well-known/openid-configuration'),
+  ssoClientId: Joi.string().required().example('ed055c9d-c3b9-449a-818e-5ffd18d49f98'),
```

Comment:
**Request change**

I know it a development app, but I don't think it's a good idea 🌝 
```suggestion
  ssoClientId: Joi.string().required().example('e39cb59e-3a74-48cc-a71d-77b8ef037ec9'),
```

Reply:
I would have sworn that I changed at leas some numbers and letters but it really is the same thing as on my db... shame on me...

Reply:
np, do you want me to remove my comment so that this is not visible anymore (you should rebase/rewrite the git history to remove the occurrence and force push) or it's not "important enough" (as it's a dev app) ?

Reply:
not important enough. It is a clientId for an environment that has two users (just you and me) and is hosted by microsoft I'm sure they have rate limiting in place to make sure they can't be bugged too much by someone requesting something with this id that they are not authorized to obtain.